device-types/HPE/ProLiant_DL380_Gen10.yaml

```diff
@@ -0,0 +1,54 @@
+---
+manufacturer: HPE
+model: ProLiant DL380 Gen10 8SFF 
+slug: hpe_dl380_g10
+part_number: P19720-B21
+u_height: 2
+is_full_depth: true
+subdevice_role: parent
+console-ports:
+  - name: Front iLO Front Service Port
+    type: usb-a
+  - name: Rear Optional Serial
+    type: de-9
+  - name: Rear VGA
+    type: other
+  - name: Front USB1
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```

Comment:
This is not the intended use of USB ports.   They should not be modelled in this fashion unless they are truely used as a console port.